### DISCUSSION TITLE
Orthovoltage backscatter simulation  EGSnrc or PENELOPE?

### DISCUSSION THREAD

**Discussion by junanzhang** [nrc-cnrc/EGSnrc#920] [Questions]
Hi Experts ,
I am planning to learn either EGSnrc or PENELOPE to simulate photon interaction inside a small animal irradiator. The goal is to calculate backscatter factor for radiobiology experiment. The voltage is 160kVp (5mm Cu HVL) or 220kVp (10mm Cu HVL). 
Should I choose EGSnrc or PENELOPE? 
Some of latest papers are using PENELOPE simulation. But I can not find any Github or active forum on PENELOPE. 
I am a little worried about the learning curve if I choose PENELOPE path. 
Any suggestions? 
Thanks
Junan

**Comment by rtownson** [comment]
For any general purpose MC code there's a steep learning curve! With EGSnrc we do have a getting-started manual to help you hit the ground running. I suggest installing EGSnrc and giving the getting-started tutorials a try, to see how it feels. Once you actually start building your geometry, feel free to reach out if you need support.

**Comment by crcrewso** [comment]
The one word of caution I would add as an admin. I havent seen much success with using Penelope in a multithreaded way. If you're simulating a lot of extremely complex work then I might lean to EGSnrc. If you can find a sample Penelope simulation of the complexity and volume that you're intending, run that and see. I personally doubt you will need that level of calculation power with what you've described, but as with most things, test it out, I am often wrong.
